x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json

```diff
@@ -413,14 +426,20 @@
       },
       "references": [],
       "type": "visualization",
-      "updated_at": "2020-05-05T00:08:18.595Z",
-      "version": "WzMyMTksMV0="
+      "updated_at": "2020-05-12T22:08:13.670Z",
+      "version": "Wzg5NiwyXQ=="
     },
     {
       "attributes": {
         "description": "",
         "kibanaSavedObjectMeta": {
-          "searchSourceJSON": {}
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
```

Comment:
Is this change necessary? I honestly don't know either way, just seems unrelated to the `interval` change.

Reply:
Good point, I just changed on the dashboard and exported the json file. I will double check and if it's not necessary, I will remove it for sure.

Reply:
Removed!